### DISCUSSION TITLE
Build for Debian 11 (bullseye)

### DISCUSSION THREAD

**Discussion by ondruska** [microsoft/openjdk#198] [Q&A] [answered]
Hello,

Debian 11 is out but the latest OpenJDK package available from Microsoft is for Debian 10 or lower. Any plans to provide updated build(s)?

Thanks,
Peter

**Comment by karianna** [comment]
One for @brunoborges after the holiday break.

**Reply by brunoborges** [reply]
Wait... What?

**Comment by karianna** [comment, marked as answer]
Hmmm actually now that I think about it - did we ever release a Debian
docker image? I thought it was just Ubuntu, Alpine and CBL-D / Mariner

On Fri, 24 Dec 2021 at 19:09, Bruno Borges ***@***.***> wrote:

> Wait... What?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/microsoft/openjdk/discussions/198#discussioncomment-1868302>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AABME2G2O3OMUNQAX4LIZPDUSTANLANCNFSM5KU2RF5Q>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
-- 
Cheers, Martijn (Sent from Gmail Mobile)

**Reply by brunoborges** [reply]
What we have is documented steps for how customers can choose whatever base OS and version they prefer.

https://docs.microsoft.com/en-gb/java/openjdk/containers#use-a-different-base-os-image

**Comment by ondruska** [comment]
Does it mean that if packaged version is not available (https://docs.microsoft.com/en-gb/java/openjdk/install#install-on-debian) I should use https://docs.microsoft.com/en-gb/java/openjdk/download-major-urls#openjdk-11-lts because distro specific packages are provided for convenience?

**Reply by brunoborges** [reply]
You can definitely use a tar.gz package and just extract it inside any Linux distro or base image.
